components/automate-chef-io/content/docs/iam-v2-overview.md

```diff
@@ -152,23 +149,105 @@ The default policies are: Viewers, Editors, Administrator, and Ingest.
 
 Just add members to any of these default policies, as described later, to quickly get your basic users up and running.
 
-### Policies and Roles in Automate
+## Working with Projects
 
-<!-- I do not like saying "user interface" because technically that includes the command line, too. But I can live with it if you think that is the lesser of two evils. -->
-IAM v2 introduces partial policy management from within Chef Automate in your browser.
+IAM projects are collections of resources created in Chef Automate or ingested from external data providers, including Chef Infra and Chef Inspec.
+Projects are used in a policy to reduce the scope of permissions of that policy to just the resources assigned to the given projects.
+Without a policy statement allowing an operation for a resource assigned to a given project, a user cannot perform the actions of that policy.
+
+{{< info >}}
+Note that Chef Automate is currently limited to six projects while we continue to refine the user experience during this beta period.
+{{< /info >}}
+
+There are four steps to setting up projects:
+
+1. Define each project via the UI (or the command line if you prefer).
+   Start on the projects list page shown previously to create your projects.
+2. Create or modify IAM policy statements to restrict permissions to specific projects as needed.
+   *Every* statement must either indicate specific projects or specify the wildcard denoting all projects (`*`).
+   By default, any previously existing policies are automatically setup with that wildcard so they apply to all projects.
+3. Assign resources with projects (see [Assigning Resources to Projects]({{< relref "iam-v2-overview.md#assigning-resources-to-projects" >}})).
+4. Select the projects to filter in the UI.
+   Once you create some projects, use the **global project filter** in the top navigation bar to select one or more projects for viewing.
+   With no selection (the default), you will see resources for *all* projects.
+
+Note that out-of-the-box we provide a *project-owner* role so the global admin may delegate much of these responsibilities for project management to others, to alleviate some of the burden for the global admin.
+
+### Properties of a Project
+
+A v2 project consists of a list of **rules**, where each rule describes a group of node characteristics.
+
+Property   | Description
+-----------|------------
+Rules      | list of project assignment rules
+
+A rule consists of a list of **conditions**, where each condition describes a single characteristic.
+It also contains a type where <<TODO describe what node and event mean>>
```

Comment:
Need someone to fill in appropriate wording here.

Reply:
node corresponds to client run nodes and compliance nodes
event corresponds to events on the event-feed page

i wouldnt use those words, but something like them